File: scripts/GAs/Experiment_Results_v1-Python-code/intermediate/SR/Translation/use_test/WizardCoder-Python-34B-V1.0/Avatar/Python/Java/codeforces_154_A.java
```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String s = scanner.nextLine();
        int n = scanner.nextInt();
        int ans = 0;
        int firstCondition = 392;
        int secondCondition = 975;
        for (int counter = 0; counter < n; counter++) {
            String x = scanner.next();
            int a = 0;
            int b = 0;
            for (int i = 0; i < s.length(); i++) {
                char j = s.charAt(i);
                if ((firstCondition & secondCondition) != 0) {
                    if (j == x.charAt(0)) {
                        a++;
                    } else if (j == x.charAt(1)) {
                        b++;
                    } else {
                        ans += Math.min(a, b);
                        a = 0;
                        b = 0;
                    }
                }
            }
            ans += Math.min(a, b);
        }
        System.out.println(ans);
    }
}
```

Test input:
mbmxuuuuxuuuuhhooooxxxuxxxuxuuxuuuxxjvjvjjjjvvvjjjjjvvjvjjjvvvjjvjjvvvjjjvjvjvjjjjjmmbmbbbbbmbbbbmm
5
mb
ho
xu
jv
yp

Expected output:
37 

File: dataset/Intermediate/Translation/Avatar/v1-Python-code/codeforces_154_A.py
import numpy as np
s = input()
n = int(input())
ans = 0
firstCondition = 392
secondCondition = 975
for counter in range(n):
    x = input()
    (a, b) = (0, 0)
    for j in s:
        if firstCondition & secondCondition:
            if j == x[0]:
                a = a + 1
            elif j == x[1]:
                b = b + 1
            else:
                ans = ans + np.min(np.array([a, b]))
                (a, b) = (0, 0)
    ans = ans + np.min(np.array([a, b]))
print(ans)